dfastmi/io/RiversObject.py

```diff
@@ -86,54 +64,54 @@ def _read_rivers_file(self, filename):
         with open(filename, "r") as configfile:
             config.read_file(configfile)
 
-        self._river_data = RiverParameterData(config)
+        river_data = RiverParameterData(config)
 
         # initialize rivers dictionary
         iversion = self._validate_version_in_file(filename, config)
 
         self._verify_checksum_rivers(config, filename)
         
         # parse branches
+        self.branches = []
         self._parse_branches(config)
 
         # parse reaches and discharge locations
-        self._parse_reaches_and_discharge_locations(config)
+        self._parse_reaches(config, iversion)
 
         # collect the values for all other quantities
-        self.nreaches = [len(x) for x in self.allreaches]
+        #self.nreaches = [len(x) for x in self.allreaches]
```

Comment:
whats the plan with this commented part?

Reply:
I have removed the comment